docs/rum_getting_started.md

```diff
@@ -42,8 +42,9 @@ dependencies {
 1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][2].
 2. Select `android` as the application type and enter an application name to generate a unique Datadog application ID and client token.
 3. To instrument your web views, click the **Instrument your webviews** toggle. For more information, see [Web View Tracking][13].
+4. To disable automatic user data collection for either client IP or geolocation data, uncheck the boxes for those settings. For more information, see [RUM Android Data Collected][15].
 
-{{< img src="real_user_monitoring/android/create_rum_application.png" alt="Create a RUM application in Datadog workflow" style="width:90%;">}}
+   {{< img src="real_user_monitoring/android/new-rum-app-android-1.png" alt="Create a RUM application in Datadog workflow" style="width:90%;">}}
```

Comment:
I get 404 when trying to open it using https://docs.datadoghq.com/real_user_monitoring/android/new-rum-app-android-1.png. The asset is not yet uploaded?